deepchem/models/torch_models/layers.py

```diff
@@ -6150,4 +6150,186 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         T_out = self.sigmoid_fn(self.T(x))
         output = H_out * T_out + x * (1 - T_out)
 
-        return output
\ No newline at end of file
+        return output
+
+def cosine_dist(x, y):
+  """Computes the inner product (cosine similarity) between two tensors.
+
+    This assumes that the two input tensors contain rows of vectors where
+    each column represents a different feature. The output tensor will have
+    elements that represent the inner product between pairs of normalized vectors
+    in the rows of `x` and `y`. The two tensors need to have the same number of
+    columns, because one cannot take the dot product between vectors of different
+    lengths. For example, in sentence similarity and sentence classification tasks,
+    the number of columns is the embedding size. In these tasks, the rows of the
+    input tensors would be different test vectors or sentences. The input tensors
+    themselves could be different batches. Using vectors or tensors of all 0s
+    should be avoided.
+
+    Methods
```

Comment:
Can you please add a screenshot of how the docs are rendering. You can find instructions to generate docs in the docs folder.